Meta/Lagom/CMakeLists.txt

```diff
@@ -197,15 +197,18 @@ function(lagom_lib library fs_name)
 endfunction()
 
 function(lagom_test source)
-    cmake_parse_arguments(LAGOM_TEST "" "" "LIBS" ${ARGN})
+    cmake_parse_arguments(LAGOM_TEST "DISABLED" "" "LIBS" ${ARGN})
     get_filename_component(name ${source} NAME_WE)
     add_executable(${name}_lagom ${source})
     set_target_properties(${name}_lagom PROPERTIES OUTPUT_NAME ${name})
     target_link_libraries(${name}_lagom LagomCore LagomTest LagomTestMain ${LAGOM_TEST_LIBS})
-    add_test(
-            NAME ${name}
-            COMMAND ${name}_lagom
-    )
+    if(${LAGOM_TEST_DISABLED})
```

Comment:
This could be done using the DISABLED test property instead
https://cmake.org/cmake/help/latest/prop_test/DISABLED.html

so something like

```
        foreach(source ${LIBMARKDOWN_TEST_SOURCES})
            lagom_test(${source} LIBS LagomMarkdown DISABLED)
        endforeach()
        set_tests_properties(TestCommonMark PROPERTIES DISABLED YES)
        # or, to do what the wasm tests do, and still run but ignore the failure (catches UBSAN violations)
        set_tests_properties(TestCommonMark PROPERTIES SKIP_RETURN_CODE 1)
```

would be a bit cleaner imo